index.d.ts

```diff
@@ -40,21 +40,21 @@ export {Entry} from './source/entry';
 export {Entries} from './source/entries';
 export {SetReturnType} from './source/set-return-type';
 export {Asyncify} from './source/asyncify';
-export {Simplify, SimplifyOptions} from './source/simplify';
+export {Simplify} from './source/simplify';
 export {Jsonify} from './source/jsonify';
 export {Schema} from './source/schema';
 export {LiteralToPrimitive} from './source/literal-to-primitive';
 export {
-	PositiveInfinity,
-	NegativeInfinity,
-	Finite,
-	Integer,
-	Float,
-	NegativeFloat,
-	Negative,
-	NonNegative,
-	NegativeInteger,
-	NonNegativeInteger,
+  PositiveInfinity,
+  NegativeInfinity,
+  Finite,
+  Integer,
+  Float,
+  NegativeFloat,
+  Negative,
+  NonNegative,
+  NegativeInteger,
+  NonNegativeInteger,
```

Comment:
Don't make unrelated changes.

Reply:
Sorry, I'm having trouble getting xo to work properly with my setup.

Reply:
@sindresorhus Done, but should `xo --fix` replace spaces with tabs? I have to change my VSCode config every time I work on type-fest (and I often forget). I have installed and configured the XO plugin and deactivated eslint and prettier. What am I doing wrong?

Reply:
> but should xo --fix replace spaces with tabs?

Yes, this project uses tab indentation.

Reply:
> Yes, this project uses tab indentation.

I known, but when I run `xo --fix` in the project it doesn't do the conversion...